docs/source/classes/Jetscape/index.rst

```diff
@@ -6,20 +6,8 @@ Jetscape
 .. currentmodule:: Jetscape
 .. autoclass:: Jetscape
 
-.. automethod:: Jetscape.particle_list
+.. automethod:: Jetscape.create_loader
 .. automethod:: Jetscape.particle_objects_list
-.. automethod:: Jetscape.num_events
-.. automethod:: Jetscape.num_output_per_event
-.. automethod:: Jetscape.get_sigmaGen
-.. automethod:: Jetscape.particle_species
-.. automethod:: Jetscape.remove_particle_species
-.. automethod:: Jetscape.lower_event_energy_cut
-.. automethod:: Jetscape.charged_particles
-.. automethod:: Jetscape.uncharged_particles
-.. automethod:: Jetscape.strange_particles
-.. automethod:: Jetscape.pt_cut
-.. automethod:: Jetscape.rapidity_cut
-.. automethod:: Jetscape.pseudorapidity_cut
-.. automethod:: Jetscape.multiplicity_cut
-.. automethod:: Jetscape.particle_status
+.. automethod:: Jetscape._particle_as_list
+.. automethod:: Jetscape.spacetime_cut
```

Comment:
Why do you keep the spacetime cut while moving all the others?

Reply:
Spacetime cut is only possible if position information is available, which is not the case for Jetscape. That's why there is an function on the level of individual storers, with one for Jetscape as well.

Reply:
Double check that all cuts are present in the base storer and marked as not implement... (way a)